Integrate esm apps

ClementChaumel opened the pull request:
## Done

- Apply IA changes to integrate ESM apps into the site

Here is a detailed list of the changes 
https://github.com/canonical-web-and-design/web-squad/issues/2346#issuecomment-721062383

~I only omitted the `/pricing and /pricing/infra` and the `legal` section because I wasn't exactly clear on what to do and was missing copy @matthewpaulthomas~


## QA

- go to https://ubuntu-com-8926.demos.haus/esm or https://ubuntu-com-8926.demos.haus/esm-apps
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the links are working correctly
- Check that all the mentions of ESM are changed to ESM Infra

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2347

webteam-app commented:
Demo starting at https://ubuntu-com-8926.demos.haus

ClementChaumel commented on templates/legal/ubuntu-advantage-service-description/index.md:
```diff
@@ -33,7 +33,7 @@ When an Ubuntu Advantage subscription is attached to a physical host running a [
   4. Specific kernel versions are not guaranteed to receive livepatches indefinitely.  
   5. Only the default LTS kernel is available for livepatching. This includes  its backport as the last HWE kernel to the previous LTS release.
-4. Extended Security Maintenance
+4. Extended Security Maintenance // what to do with the legal text?
   1. Extended Security Maintenance provides available High and Critical CVE fixes for a number of server packages. A complete list of packages included in Extended Security Maintenance for a given release can be found at: [https://wiki.ubuntu.com/SecurityTeam/ESM/](https://wiki.ubuntu.com/SecurityTeam/ESM/)
   2. Extended Security Maintenance is only included for 64-bit x86 AMD/Intel installations.
   3. Extended Security Maintenance does not provide:
```
Here I'm missing the updated legal text

ClementChaumel commented on templates/shared/pricing/_ua-for-infrastructure-ja.html:
```diff
@@ -65,7 +65,7 @@
         </tr>
         <tr>
-          <td><a href="/esm">拡張セキュリティメンテナンス</a> (ESM)</td>
+          <td><a href="/esm-infra">拡張セキュリティメンテナンス</a> (ESM Infra)</td>
```
Do we need Japanese translation here?

anasereijo commented:
@ClementChaumel this is looking good! 

As you are making changes to /esm-infra page, can you also make [these small changes](https://github.com/canonical-web-and-design/ubuntu.com/issues/8933)? It's just updating one illustration and deleting an icon. Thanks!

anasereijo commented:
@ClementChaumel ignore my request about updating the illustrations. @bethcollins92 already [done it ](https://github.com/canonical-web-and-design/ubuntu.com/pull/8938)

matthewpaulthomas reviewed (changes requested):
Checking against [the design](https://github.com/canonical-web-and-design/web-squad/issues/2346#issuecomment-721062383):
✓ “Enterprise” mega-menu
✓ “Security” ribbon
✓ Footer sitemap
✓ /esm
✓ /support
✓ /pricing
✗ /pricing/infra — changes not made
⋯ /legal/ubuntu-advantage — TBD
✓ /about/release-cycle

Well done on such a thorough sweep of the site! I’ve found a few things changed, though, that should probably stay unchanged:
- the ESM Infra page URL
- the names of ESM releases (e.g. “14.04 ESM”, not “14.04 ESM Infra”)
- prose which will now be referring to ESM in general, rather than ESM Infra in particular.

matthewpaulthomas commented on templates/download/server/download.html:
```diff
@@ -73,7 +73,7 @@ <h2 class="p-heading--three">Get Ubuntu Server pro tips</h2>
       <div class="col-6">
         <h3>Enterprise Support for Ubuntu Server</h3>
-        <p>Looking for commercial support for Ubuntu Server? Ubuntu Advantage for Infrastructure (UA-I) provides production-grade SLAs, phone and ticket support, Extended Security Maintenance (ESM) and more.</p>
+        <p>Looking for commercial support for Ubuntu Server? Ubuntu Advantage for Infrastructure (UA-I) provides production-grade SLAs, phone and ticket support, Extended Security Maintenance for Infrastructure (ESM Infra) and more.</p>
```
I missed this one in my drive-by:
```suggestion
        <p>Looking for commercial support for Ubuntu Server? Ubuntu Advantage for Infrastructure (UA Infra) provides production-grade SLAs, phone and ticket support, Extended Security Maintenance for Infrastructure (ESM Infra) and more.</p>
```

ClementChaumel commented on navigation.yaml:
```diff
@@ -1265,8 +1265,10 @@ security:
-    - title: ESM
-      path: /security/esm
+    - title: ESM Infra
+      path: /security/esm-infra
```
Ok that makes sense. How about the `esm-apps` url is it fine that way or do you want it named something else? could even go under `esm/apps`

matthewpaulthomas replied:
I think esm-apps is fine.

ClementChaumel commented:
I've added the changes to /pricing and /pricing/infra. I've also updated the comparison table and made it so /advantage and /pricing/infra use the same template

albertkol commented on templates/legal/ubuntu-advantage-service-description/ja/index.md:
```diff
@@ -12,6 +12,8 @@ context:
 
 {% include "shared/pricing/_ua-for-infrastructure-ja.html" %}
 
+<!-- TODO is the new table ok for the legal section? + Needs Japanese translation -->
```
I don't think questions should be in the code base.

I see 4 TODOs in the code. Why is it? Are you still waiting for answers? Or are we planning to address them later? If the plan is to deal with them later then we should instead have an issue explaining what is needed instead of comments.

ClementChaumel replied:
This PR going on a feature branch it was more to keep track of what was missing. I planned to make sure all the todos were gone before merging to master. You're right I should create issues!

albertkol commented on templates/pricing/infra.html:
```diff
@@ -11,31 +11,29 @@
-      </li>
-    </ul>
+    <h1>Support and security pricing</h1>
+    <!-- TODO Find copy -->
```
Did you find it? :P

ClementChaumel replied:
As I understand it it's not written yet. so... no!

albertkol reviewed (commented):
Comments on comments.

matthewpaulthomas commented on templates/shared/pricing/_maas-pricing.html:
```diff
@@ -24,7 +24,7 @@
         <tr>
           <td>Response time - SLA Sev 1</td>
-          <td class="u-align--center">-</td>
+          <td class="u-align--center"><span alt="No">—</span></td>
```
```suggestion
          <td class="u-align--center" aria-label="No">—</td>
```
`alt` is valid for `area`, `img`, or `input`, not `span`. I think you’re right, though, that accessibility requires explaining what the dash means. This is a rare case of an appropriate use for `aria-label`.

matthewpaulthomas commented on templates/shared/pricing/_ua-for-infrastructure.html:
```diff
@@ -1,151 +1,119 @@
+        <tr id="esm-apps">
+          <td>ESM for non-base packages ("Universe") in Ubuntu 14.04, 16.04, 18.04, and 20.04 LTS</td>
+          <td class="u-align--center"><img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes" width="16" height="16"></td>
+          <td class="u-align--center"><span alt="No">—</span></td>
```
All these `span alt`s should similarly be `td aria-label`s.

matthewpaulthomas reviewed (changes requested):
Today Lech has confirmed [what he mentioned after we’d finished the IA design last month](https://github.com/canonical-web-and-design/web-squad/issues/2346#issuecomment-754758893): there will be just one ESM marketing page, covering both ESM Infra and ESM Apps, not separate pages as we’d assumed.

Unfortunately, that requires quite a few changes to this PR. (Sorry Clement!)

cristinadresch commented:
Closing it for now since it's not relevant anymore
